tests/plugins/easyimage/uploadintegrations.js

```diff
@@ -245,6 +247,29 @@
 
 				easyImageDef.loaderType = AsyncFailFileLoader;
 
+				assertPasteFiles( this.editor, {
+					files: [ bender.tools.getTestPngFile() ],
+					fullLoad: true,
+					callback: function( widgets ) {
+						easyImageDef.loaderType = originalLoader;
+
+						assert.areSame( 1, window.alert.callCount, 'Alert call count' );
+						sinon.assert.alwaysCalledWith( window.alert, 'Input buffer contains unsupported image format.' );
```

Comment:
We don't usually use `sinon.assert`. It would be probably better to use something like this:

```javascript
assert.isTrue( window.alert.alwaysCalledWith( 'Input buffer contains unsupported image format.' ) );
```